services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -72,7 +72,7 @@ export default class CurrentSubscription extends React.Component {
     return (
       <span>
         To renew your subscription for next year, contact us now
-        at <a href="mailto:sales@quill.org">sales@quill.org</a>.
+        at <a href="https://quill.org/request_renewal">sales@quill.org</a>.
       </span>
```

Comment:
I think we'd probably want to change this to maybe have "contact us now" be the displayed link text